Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentStore.cs

```diff
@@ -523,6 +523,27 @@ BoolResult waitForCompletion()
             }
         }
 
+        /// <summary>
+        /// Attempts to get local location store if enabled
+        /// </summary>
+        public bool TryGetLocalLocationStore(out LocalLocationStore localLocationStore)
+        {
+            if (_contentLocationStore is TransitioningContentLocationStore tcs

```

Comment:
It is interesting that TryGetLocalLocationStore is quite different from LocalLocationStore property.

One may think that these two are the same and one is implemented on top of another. But only the 'TryGet' one will return false in case if the store is disabled.